frontend_lib/src/css/IconButton.styl

```
@@ -33,3 +34,5 @@
   &__secondary_dark
     border-color: var(--primaryDarkColor)
     border-width: 1px
+  &__label
```

```suggestion
  &__text
```
as the property is named `text`.